OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/cg/xta/TypePropagationAnalysis.scala

```diff
@@ -268,7 +268,9 @@ final class TypePropagationAnalysis private[analyses] (
             // Instances of DefinedMethod we see should only be those where the method is defined in the class file of
             // the declaring class type (i.e., it is not a DefinedMethod instance of some inherited method).
```

Comment:
Shouldn't we technically also adapt this comment, as it is no longer true in the exact scenario you created this PR for?

Reply:
I merged the other comment into this one to hopefully improve the clarity, let me know what you think.